CriticalMass/CMLogoActivityView.swift

```diff
@@ -0,0 +1,426 @@
+//
+//  CriticalMapsActivityView.swift
+//  CriticalMaps
+//
+//  Created by Malte Bünz on 20.07.19.
+//  Copyright © 2019 Pokus Labs. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+@IBDesignable public class CMLogoActivityView: UIView, IBConstructable {
+    private var timer: Timer?
+    private var angle: CGFloat = 0
+
+    public override func draw(_: CGRect) {
+        CriticalMapsLoader.drawCMLogo(rotation: angle)
+    }
+
+    public func stopAnimating() {
+        timer?.invalidate()
+        timer = nil
+        angle = 0
+        setNeedsDisplay()
+    }
+
+    public func startAnimating() {
+        if timer == nil {
+            timer = Timer.scheduledTimer(timeInterval: 0.04,
```

Comment:
Why are you using 0.04 here? CADisplayLink might be better here instead of hardcoding a framerate

Reply:
Thats a good idea. Will use that